### ISSUE TITLE
508-defect-1 ❗ Launchblocker [AXE-CORE]: Error id attribute value must be unique

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#19790]
# [508-defect-1 ❗ Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## Axe Violation

**[ID attribute values must be unique - Minor](https://dequeuniversity.com/rules/axe/3.5/duplicate-id?application=coconut)**
The ID attribute uniquely identifies elements on a page. It does not make sense to duplicate an ID.

Duplicate ID's can break the accessibility of labels for forms, table header cells, etc., by the second instance being skipped by screen readers, or by client-side scripts. They are common markup validation errors that can eliminate possible sources of accessibility problems, when not breaking accessibility.

Those experienced with client-side scripting know that when you re-use an id, typically the only one that gets acted upon by the scripting is the first instance of the use of that ID. Similarly, assistive technologies may, when referencing an id, only reference the first one accurately.

## Acceptance Criteria

- [ ] Axe no longer flags a violation for the error message

## Steps to Recreate

1. Recreate the error by editing and saving the first account
2. Run axe scan
3. Confirm "ID attribute values must be unique" is flagged for the error message

## Proposed Solution

Use a unique id attribute value

## Screenshots or Trace Logs

<img width="1735" alt="Screen Shot 2021-02-10 at 3 41 29 PM" src="https://user-images.githubusercontent.com/14154792/107570142-25112580-6bb7-11eb-8861-b2e6c43ef75c.png">
<img width="1769" alt="Screen Shot 2021-02-10 at 3 49 16 PM" src="https://user-images.githubusercontent.com/14154792/107570500-9b158c80-6bb7-11eb-91ef-acfd8bb3510a.png">

**Comment by 1Copenut** [comment]
@joshkimux This should be escalated to a `508-defect-1` unless the product team is confident it'll get fixed with work for #19594. Our apps are required to have axe checks in the end to end tests per the ATO memo on accessibility. There might be a need to boost the number of axe checks in the profile app. I'll mention that during the staging review tomorrow after we've discussed these launch blocking issues with the team.

**Comment by joshkimux** [comment]
Updated to 508-defect-1!

**Comment by erikphansen** [comment]
@joshkimux the steps to repro say you only need to trigger an error when saving bank info for the first feature. Can you confirm that you need to trigger errors for _both_ direct deposit features for this duplicate ID issue to occur?

**Comment by erikphansen** [comment]
Should be easy to fix by updating the wrapper `div`s:
https://github.com/department-of-veterans-affairs/vets-website/blob/566cbc2a7b3acb6afc35b248aab6cf47515b4571/src/applications/personalization/profile/components/direct-deposit/BankInfoEDU.jsx#L190
https://github.com/department-of-veterans-affairs/vets-website/blob/566cbc2a7b3acb6afc35b248aab6cf47515b4571/src/applications/personalization/profile/components/direct-deposit/BankInfoCNPv2.jsx#L220
